src/flask_debugtoolbar/__init__.py

```diff
@@ -3,15 +3,8 @@
 import warnings
 
 import flask
-from packaging import version as version_builder
 from flask import Blueprint, current_app, request, g, send_from_directory, url_for
-
-
-if version_builder.parse(flask.__version__) >= version_builder.parse("2.2.0"):
```

Comment:
Technically these code changes only require us to bump to `2.2.0` minimum, so I'm also okay if we go with that.

But I thought for a debug tool that gets minimal maintenance, life might be simpler if we only support the latest mainline Flask release rather than trying to help users debug issues with older versions.

Very okay either way and open to feedback from others...